src/components/ShopPage/ShopPageHero/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import Rating from "react-rating";
+import Link from 'next/link';
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { toaster } from 'evergreen-ui';
+
+import { EmptyStar, FullStar } from "components/StarRating";
+import { sendEvent } from 'hooks/amplitude';
+import styles from "../../BeachPageHero/styles.module.css";
+
+const ShopPageHero = (props: any) => {
+    const onWebsiteClick = () => {
+        // sendEvent('entry_map__click')
+        if (props.shop.url) { return }
+        toaster.danger(`Sorry, we don\'t have a website for this shop yet!`)
+    }
+
+    const onDirectionsClick = () => {
+        // sendEvent('directions__click')
+        if (props.shop.entry_map) { return }
```

Comment:
this is the wrong attribute, we need to construct a google maps link for this based on the address